src/components/TransactionDetails/NamespaceRegistration/NamespaceRegistration.vue

```diff
@@ -38,13 +40,13 @@ export default class NamespaceRegistration extends Vue {
       'registrationType',
     )
     const duration: UInt64 = this.view.values.get('duration')
-
+    const blockGenerationTargetTime = this.networkConfiguration.blockGenerationTargetTime
```

Comment:
Where is `this.networkConfiguration` model populated with a value? I don't see it mapped from Store.

Reply:
Missed:?
```
computed: {
    ...mapGetters({
      networkConfiguration: 'network/networkConfiguration',
    }),
  },
```